src/datasources/alarms/types/ListAlarms.types.ts

```diff
@@ -1,8 +1,9 @@
-import { AlarmsQuery } from './types';
+import { AlarmsQuery, TransitionInclusionOption } from './types';
 
 export interface ListAlarmsQuery extends AlarmsQuery {
   filter?: string;
   properties?: AlarmsProperties[];
+  transition?: TransitionInclusionOption;
```

Comment:
```suggestion
  transitionInclusionOption?: TransitionInclusionOption;
```
Having it as transition does not give proper meaning